### PR TITLE
Copy news items from old quattor sourceforge into current format

### PR DESCRIPTION
**Pull request by GooBall**
As requested by @jrha.

### CODE REVIEW

File: _posts/2009-02-27-slides-posted-from-recent-tutorial.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+---
+layout: article
+title: Slides posted from recent tutorial
+category: news
+author: Nick Williams
+---
+
+Recently, a tutorial was run in France for advanced users of quattor. The slides are now available online via the links at:
+
+http://apps.sourceforge.net/mediawiki/quattor/index.php?title=For_Users#Tutorials
```
Comment (jrha) [inline _posts/2009-02-27-slides-posted-from-recent-tutorial.md]: Dead link :disappointed:

Comment (jouvin) [inline _posts/2009-02-27-slides-posted-from-recent-tutorial.md]: I'll see if I find the real link...

**Comment by wdpypere** [comment]
except for the dead link comment, LGTM.

**Comment by wdpypere** [comment]
agreed with @jrha we will replace the link in a separate PR.

**Pull request merged**